polaris/hub/polarisfs.py

```diff
@@ -105,6 +124,14 @@ def ls(
             for p in response.json()
         ]
 
+        if self.use_listings_cache:
+            self.dircache[path] = detailed_listings
```

Comment:
`self.use_listings_cache` should always exist for us, so we can simplify this to:

```suggestion
        self.dircache[path] = detailed_listings
```